### ISSUE TITLE
If LDAP_ROOT contains spaces custom ldif files can't be used for bootstrapping

### ISSUE THREAD

**Issue by christian-scheible** [bitnami/bitnami-docker-openldap#55]
**Description**

If LDAP_ROOT contains spaces custom ldif files can't be used for bootstrapping as ldapadd fails due to a space in the bind DN.

The problem can be fixed using quotes for LDAP_ADMIN_DN on this line:
https://github.com/bitnami/bitnami-docker-openldap/blob/master/2.5/debian-10/rootfs/opt/bitnami/scripts/libopenldap.sh#L418

**Steps to reproduce the issue:**

1. Set LDAP_ROOT to something like `LDAP_ROOT: "o=some org,c=com"`
2. Bootstrap LDAP using ldif files in the LDAP_CUSTOM_LDIF_DIR
3. Start container

**Describe the results you received:**

Start of container fails with: `exited with code 123`

**Describe the results you expected:**

Start of container doesn't fail with error message.

**Additional information you deem important (e.g. issue happens only occasionally):**

This is my docker-compose file:

```
version: '3.7'

services:

  ldap:
    image: bitnami/openldap:2.4
    volumes: 
      - ./bootstrap:/ldifs
    environment:
      LDAP_ROOT: "o=some org,c=de"
```
ldif file needs to be in the bootstrap folder.

**Version**

- Output of `docker version`:

```
Client:
 Version:           20.10.8
 API version:       1.41
 Go version:        go1.16.6
 Git commit:        3967b7d28e
 Built:             Wed Aug  4 10:59:01 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.8
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.6
  Git commit:       75249d88bc
  Built:            Wed Aug  4 10:58:48 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.5.5
  GitCommit:        72cec4be58a9eb6b2910f5d10f1c01ca47d231c0.m
 runc:
  Version:          1.0.2
  GitCommit:        v1.0.2-0-g52b36a2d
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

```

- Output of `docker info`:

```
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Build with BuildKit (Docker Inc., v0.6.1-docker)

Server:
 Containers: 16
  Running: 4
  Paused: 0
  Stopped: 12
 Images: 130
 Server Version: 20.10.8
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: false
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 72cec4be58a9eb6b2910f5d10f1c01ca47d231c0.m
 runc version: v1.0.2-0-g52b36a2d
 init version: de40ad0
 Security Options:
  seccomp
   Profile: default
  cgroupns
 Kernel Version: 5.10.68-1-MANJARO
 Operating System: Manjaro Linux
 OSType: linux
 Architecture: x86_64
 CPUs: 8
 Total Memory: 15.28GiB
 Name: christian-lifebooku7410
 ID: S2UC:R2BE:N3O5:CGH6:WYPL:GEED:O7JQ:D5JE:ELZC:JEKY:52LI:TBKC
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.29.2, build unknown
docker-py version: 5.0.2
CPython version: 3.9.7
OpenSSL version: OpenSSL 1.1.1l  24 Aug 2021

```

**Comment by juan131** [comment]
Thanks so much for reporting this bug! Checking your PR...

**Comment by juan131** [comment]
We just released two image revisions (`2.4.59-debian-10-r74` and `2.5.7-debian-10-r44`) including your fix @christian-scheible, see:

- https://github.com/bitnami/bitnami-docker-openldap/commit/1e2649ae25d688dcbbb00b114686f343014396aa
- https://github.com/bitnami/bitnami-docker-openldap/commit/2694704a008c22f84ab10cf7ab5b1c254df4247f

Thanks again for reporting this and proposing a solution!!

**Comment by christian-scheible** [comment]
Thanks, @juan131, for this quick release.
